README.md

```diff
@@ -187,6 +186,28 @@ services:
     enabled: true
 ```
 
+To enable Keel, ensure that `managedDelivery.enabled` is set to `true` in your hal
+config, and then ensure the Keel kustomization is included in the `resources`
+block of your `kustomization.yml`:
+
+```
+resources:
+- github.com/spinnaker/kustomization-base/core
+- github.com/spinnaker/kustomization-base/keel
+```
+
+
+Since the default
+[service discovery file](/core/service-discovery/spinnaker.yml) defaults
+Keel to disabled, you will need to configure a service discovery overrides file
+in your fork of spinnaker-config with the following block:
+
+```
+services:
+  keel:
+    enabled: true
```

Comment:
Did we determine that we don't actually need this and `kleat` will write out the `enabled` block when managed delivery is enabled in users' halconfigs?

Reply:
yeah I believe so. I was waiting on answer to your question from @ezimanyi on the other thread before nuking it for Keel and Kayenta.

Reply:
Thanks for updating this!